### ISSUE TITLE
working path can't contain a comma

### ISSUE THREAD

**Issue by ren0x** [unyt-org/uix#46]
`uix` crashes when the working path contains a `,` (comma), e.g. `.../test, test/...`

Executing `~/Projects/test, test$ uix` results in:
```bash
error: Uncaught (in promise) SyntaxError: Invalid unit symbol 'test'
            if (prefix_exp == null) throw new SyntaxError("Invalid unit symbol '" + prefixed_symbol + "'")
                                          ^
    at Quantity.getExpandedSymbolWithPrefix (https://dev.cdn.unyt.org/unyt_core/types/quantity.ts:405:43)
    at Quantity.parseUnit (https://dev.cdn.unyt.org/unyt_core/types/quantity.ts:368:42)
    at new Quantity (https://dev.cdn.unyt.org/unyt_core/types/quantity.ts:118:57)
    at Function.parseNextExpression (https://dev.cdn.unyt.org/unyt_core/compiler/compiler.ts:4567:46)
    at Function.compileLoop (https://dev.cdn.unyt.org/unyt_core/compiler/compiler.ts:4959:24)
    at async Function.executeDatexLocally (https://dev.cdn.unyt.org/unyt_core/runtime/runtime.ts:809:56)
    at async get (https://dev.cdn.unyt.org/unyt_core/datex_short.ts:77:21)
    at async getAppOptions (https://dev.cdn.unyt.org/uix/app/config-files.ts:25:22)
    at async https://dev.cdn.unyt.org/uix/run.ts:103:59
```

Although this is a rather unusual/undesirable folder name, it should not be a problem.

**Comment by benStre** [comment]
This was actually caused by a DATEX injection vulnerability in `datex.get` (could lead to arbitrary DATEX code execution if you name your file paths the "right" way)

Lesson learned: make sure input sanitation is used everywhere :)

**Comment by benStre** [comment]
Fixed in [datex-core-js@0.0.16](https://github.com/unyt-org/datex-core-js-legacy/releases/tag/v0.0.16) / [uix@0.1.6](https://github.com/unyt-org/uix/releases/tag/v0.1.6)
